app/components/attachment_button/index.js

```diff
@@ -178,6 +179,7 @@ export default class AttachmentButton extends PureComponent {
 
         if (hasCameraPermission) {
             ImagePicker.launchCamera(options, (response) => {
+                StatusBar.setHidden(false);
```

Comment:
This will hide the status bar even in portrait mode

Reply:
This sets hidden to *false*. The problem was that the StatusBar was hidden (or not recognized / safe area affected) when returning to Portrait mode. What case are you finding the status bar hidden in portrait mode?